desktop_version/src/Script.cpp

```diff
@@ -448,7 +448,14 @@ void scriptclass::run(void)
             }
             if (words[0] == "playef")
             {
-                music.playef(ss_toi(words[1]));
+                if (music.soundidexists(words[1].c_str()))
+                {
+                    music.playefid(words[1].c_str());
+                }
+                else if (!music.soundisextra(ss_toi(words[1])))
+                {
+                    music.playef(ss_toi(words[1]));
+                }
```

Comment:
Should we warn if a sound was requested and not found at all?

Reply:
I tend to not leave warnings for scripting, however maybe we should -- there should most likely be a better place to warn about script errors other than the console however, but I can add a warning here for now.